mpris.c

```diff
@@ -0,0 +1,558 @@
+/*
+ * Copyright 2016 Various Authors
```

Comment:
These lines have no legal effect. All they do is advertise who wrote the code. And since most cmus files simply use "Various Authors" I'm fine with it here too.